collections/hub/golden_path/settlement_management/mixed_settlement_model.json

```
@@ -3377,13 +3415,23 @@
             "postRequest": {
               "exec": [
                 "if (environment.ENABLE_WS_ASSERTIONS===true) {",
-                "  environment.testfsp2callbackst = await websocket.getMessage('testfsp2callbackst', environment.WS_ASSERTION_TIMEOUT)",
+                "  // environment.testfsp2callbackst = await websocket.getMessage('testfsp2callbackst', environment.WS_ASSERTION_TIMEOUT)",
```

1. Shouldn't we comment out the preRequest (https://github.com/mojaloop/testing-toolkit-test-cases/pull/129/files#diff-3c071aa2e0e971ce0363ecad2a443b8fc7ba6afcb1b44c962e21a514df60a396R3411)?

2. See suggested changes

```suggestion
                "  // NOTE: This has been commented out due to the WS issue https://github.com/mojaloop/project/issues/3164, with a work-around to call the HTTP equivalent API on the postRequest instead until it can be resolved.",
                "  // environment.testfsp2callbackst = await websocket.getMessage('testfsp2callbackst', environment.WS_ASSERTION_TIMEOUT)",
```